pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/proj/dao/StockDao.java

```diff
@@ -0,0 +1,126 @@
+package com.subho.wipro.pjp.tm08.proj.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import com.subho.wipro.pjp.tm08.proj.bean.Product;
+import com.subho.wipro.pjp.tm08.proj.util.DBUtil;
+
+public class StockDao {
+	
+	public int insertStock(Product product) {
+		Connection conn = null;
+		PreparedStatement pstmt = null;
+		String sql = "INSERT INTO TBL_STOCK VALUES(?, ?, ?, ?, ?)";
+		
+		try {
+			conn = DBUtil.getConnection();
+			pstmt = conn.prepareStatement(sql);
+			pstmt.setString(1,product.getProductID());
+			pstmt.setString(2, product.getProductName());
+			pstmt.setInt(3, product.getQuantityOnHand());
+			pstmt.setDouble(4, product.getProductUnitPrice());
+			pstmt.setInt(5, product.getReorderLevel());
+			
+			if (pstmt.executeUpdate() == 1) return 1;
+			else return 0;
+		} catch (SQLException e) {
+			e.printStackTrace();
+			return 0;
+		}
+	}
+
+	public String generateProductID(String productName) {
+		Connection conn = null;
+		PreparedStatement pstmt = null;
+		String sql = "SELECT SEQ_PRODUCT_ID.NEXTVAL FROM DUAL";
+		
+		int SEQ_PRODUCT_ID = 0;
+		String out = "";
+		
+		try {
+			conn = DBUtil.getConnection();
+			pstmt = conn.prepareStatement(sql);
+			ResultSet rs = pstmt.executeQuery();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)